### ISSUE TITLE
#559 broke page with parameters

### ISSUE THREAD

**Issue by mkellerman** [ironmansoftware/universal-dashboard#657]
Since the upgrade to 2.3.1, the pages with parameters stopped working.

Even the example provided doesn't seem to work:

```powershell
$Pages = @(
    New-UDPage -URL '/Parent'       -Endpoint { New-UDParagraph -Text 'Parent' }
    New-UDPage -URL '/Parent/Child' -Endpoint { New-UDParagraph -Text 'Child'  } )

Start-UDDashboard -Dashboard ( New-UDDashboard -Pages $Pages ) -Name Test1 -Port 1111
```

**Comment by mkellerman** [comment]
Not only does this 'Child' page not even load, but all pages with parameters fail to load as well:
```powershell
New-UDPage -URL '/Settings/:id'       -Endpoint { param(id) New-UDParagraph -Text "Settings: ${id}" }
```

**Comment by adamdriscoll** [comment]
This should already be resolved in 2.3.2 kits. It's due out in a day or so after I validate the fixes.

**Comment by adamdriscoll** [comment]
Here's the latest UD Enterprise build for 2.3.2: https://adamdriscollstorage.blob.core.windows.net/universal-dashboard-nightly/UniversalDashboard.857.zip
